src/swap/SwapScreenWithBack.tsx

```diff
@@ -0,0 +1,13 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
```

Comment:
similar comment as above, add a single test for this validating the drawer is not rendered and nav is..